bin/publish-node.sh

```diff
@@ -7,7 +7,7 @@ PUBLISH_NODE_TTL=${PUBLISH_NODE_TTL:-120}
 PUBLISH_NODE_POLL=${PUBLISH_NODE_POLL:-60s}
 
 NODE=$(cat /var/lib/rabbitmq/nodename)
-HOST_IP=${HOST_IP:-$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}')}
+HOST_IP=${HOST_IP:-$(hostname -I | tr -d '[[:space:]]')}
```

Comment:
I like the idea of using hostname.  However,  should we just pick the first host 
```HOST_IP=${HOST_IP:$(hostname -I | awk '{print $1}')}``` ?

The only case I could see multiple IP address is when we use --net=host. Even in that case, we might just pick the first IP in the list by default.  Thoughts ?